lib/Command/ConfigReport.php

```diff
@@ -30,8 +30,25 @@
  */
 class ConfigReport extends Command {
 
-	public function __construct() {
+	/**
+	 * @var ReportDataCollector
+	 */
+	private $reportDataCollector;
 
+	public function __construct() {
```

Comment:
@DeepDiver1975 I hope the command is only instantiated when called, not always. Else it would create the full report for nothing just when instantiating. (if the DI stuff I mentioned above is done)